docs/Lexicon.md

```diff
@@ -162,6 +162,25 @@ written "dupe". Pronounced the same way as the first syllable of
 A value whose type is a protocol composition (including a single protocol
 and *zero* protocols; the latter is the `Any` type).
 
+## explicit module build
+
+A module build where all dependency modules are passed to the compiler
+explicitly by an external build system, including any modules in caches.
+See also: [implicit module build](#implicit-module-build) and
+[fast dependency scanner](#fast-dependency-scanner).
+
+## fast dependency scanner
+
+A Swift compiler mode that scans a Swift module for import declarations and
+resolves which modules will be loaded. It is based on the
+[clang-scan-deps](https://llvm.org/devmtg/2019-04/slides/TechTalk-Lorenz-clang-scan-deps_Fast_dependency_scanning_for_explicit_modules.pdf)
+library within Clang, for (Objective-)C modules, but is extended to also
+understand textual Swift modules (.swiftinterface files).
```

Comment:
It may worth mentioning swift-driver provides APIs to digest the output of the scanner. This is to allow build systems to share more module building logics.